commands/sync/sync.sh

```diff
@@ -1,10 +1,32 @@
 #!/usr/bin/env bash
 
 set -eu -o pipefail
-shopt -s inherit_errexit
 
 . "$(dirname "${BASH_SOURCE[0]}")/../cmd_runner.sh"
 
+# Function to check syncing status
+check_syncing() {
+  # Send the system_health request and parse the isSyncing field
+  RESPONSE=$(curl --silent --request POST http://127.0.0.1:9944 \
```

Comment:
One more thing to consider, is adding `--show-error` flag, to actually see what kind of an error we're getting.  
This will go to stderr, so `RESPONSE` won't be affected

Reply:
Done, added, thank you!